option optimize_for = LITE_RUNTIME;

message NianGetRedPacketRequest
{
 required uint32 id = 1;
}
